src/routes/merge-lock.js

```diff
@@ -19,7 +19,7 @@ export function validateRepo(req, res, next) {
 
 export async function putMergeLock(req, res) {
 	const repo = req.query.repo;
-	const message = req.query.message ? `: ${req.query.message}` : '';
+	const message = req.query.message ? `: ${decodeURIComponent(req.query.message)}` : '';
```

Comment:
Hm, interesting, I thought I tested it with a properly encoded and decoded message. But I guess it makes sense this is encoded at this point